customize.dist/translations/messages.de.js

```diff
@@ -0,0 +1,230 @@
+  define(function () {
+      var out = {};
+
+      // translations must set this key for their language to be available in
+      // the language dropdowns that are shown throughout Cryptpad's interface
+      out._languageName = 'German';
+
+      out.main_title = "Cryptpad: Echtzeitzusammenarbeit, ohne Vorwissen";
+      out.main_slogan = "Einigkeit macht stark - Zusammenarbeit ist der Schlüssel"; // Der Slogan sollte evtl. besser englisch bleiben.
```

Comment:
So far cryptpad has mainly been translated to latin languages (French and Spanish). In my opinion there's a little difference between the culture of translation in these languages and German: I think we tend to keep slogans like this rather in the English original since its going to be understood by almost everyone. For my ears the German translation sounds rather funny. And that effect seems to be greater than the translation transmitting the message - although I stayed as close as possible to the English original.